### PR TITLE
Skip the heavy part of fs_stress because of poor performance on RPi

### PR DESCRIPTION
**Pull request by Zaoliang**
see https://progress.opensuse.org/issues/154417

### CODE REVIEW

**Comment by Zaoliang** [comment]
VR:

https://openqa.suse.de/tests/14196755#step/fs_stress/20

**Review by dzedro — APPROVED** [review]
If that is the only failing `file_copy` then fine ?
Otherwise I would ask if it make sense to run fs_stress on RPi

File: tests/console/fs_stress.pm [hunk 1]
```diff
@@ -25,8 +25,13 @@ sub run {
...
     upload_logs('/tmp/file_copy_500.log');
-    assert_script_run("time ./file_copy -j 4 -i 5 -s 5000 | tee /tmp/file_copy_5000.log", timeout => 1200);
-    upload_logs('/tmp/file_copy_5000.log');
+    if (check_var('FLAVOR', 'JeOS-for-RaspberryPi')) {
+        record_soft_failure('poo#154417, very poor performance on RPi, it fails even if timeout increases to 2400');
+    }
+    else {
+        assert_script_run("time ./file_copy -j 4 -i 5 -s 5000 | tee /tmp/file_copy_5000.log", timeout => 1200);
+        upload_logs('/tmp/file_copy_5000.log');
+    }
```
Comment (rfan1) [inline tests/console/fs_stress.pm:34]: Suggestion:
`return record_soft_failure 'poo#154417, very poor performance on RPi, it fails even if timeout increases to 2400' if check_var('FLAVOR', 'JeOS-for-RaspberryPi');`
assert_script_run("time ./file_copy -j 4 -i 5 -s 5000 | tee /tmp/file_copy_5000.log", timeout => 1200);
upload_logs('/tmp/file_copy_5000.log');`

Comment (Zaoliang) [reply tests/console/fs_stress.pm]: thanks!

**Comment by Zaoliang** [comment]
well, the last part of fs_stress is too much for RPi :)

**Comment by Zaoliang** [comment]
new VR: https://openqa.suse.de/tests/14212731#step/fs_stress/20
passed now

**Pull request merged**